plugins/experimental/stream_editor/stream_editor.cc

```diff
@@ -223,6 +223,7 @@ class rxscope : public scope_t
     if (error) {
       TSError("stream-editor: can't compile regexp [%s]", str);
       TSfree(str);
+      throw std::runtime_error("stream editor: Error compiling regex, regcomp in rxscope");
```

Comment:
You need `#include <stdexcept>` to define std::run_time, so I'm guess this won't compile.

Reply:
it built for me

Reply:
It must be included indirectly, probably by regex.h.  I recommend including it directly.